tools/server/pom.xml

```diff
@@ -113,6 +85,12 @@
 			<artifactId>opencsv</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.github.mjeanroy</groupId>
```

Comment:
Licensing could potentially be an issue. This is using the maven test scope, so that means that a superficial license check is all that is needed.

Reply:
Is MIT license ok? https://github.com/mjeanroy/junit-servers/blob/master/LICENSE

Reply:
That should be fine (doesn't hurt to double-check with the dash-license tool). 

On a related note, I think the dash-license tool now has a maven plugin that we can add into our workflow so these things are automatically checked and reported. https://github.com/eclipse/dash-licenses

Reply:
It seems like the dash maven plugin is not published anywhere, so it has to be installed locally. But that is a minor issue. I can check it, or does it have to be done by the project maintainers?

Reply:
The new license-check workflow is green: https://github.com/eclipse/rdf4j/pull/4437/checks

But it probably doesn't check test-scoped dependencies, or what? The introduced dependency is not listed in the summary. Anything more to do regarding this? I'm planning to move this POC into a proper PR, ref. https://github.com/eclipse/rdf4j/issues/4454